### PR TITLE
Cleanup repository structure

### LINKED ISSUE #38: Clean up repository structure
In view of https://github.com/pendulum-chain/pendulum-ink-wrapper/issues/1#issuecomment-1863207654 we should:
* rename this repository (not mentioning ink anymore)
* clean up the content, e.g., remove `price_feed` and `psp_pendulum` folders (they can go to a different branch for example or we remove them entirely)
* also check whether the `docs` folder needs to be updated

### PR DESCRIPTION
**Pull request by gianfra-t**
Closes #38.

- Remove old ink wrapper contracts (`price_feed`, `psp_pendulum`). These contracts can now be found in the branch [old-ink-wrappers](https://github.com/pendulum-chain/pendulum-ink-wrapper/tree/old-ink-wrappers).

- Update README.

### CODE REVIEW

**Comment by gianfra-t** [comment]
We still need to rename the repository, would `Pendulum-Solidity Wrappers` or `Pendulum-Solidity Wrapper Contracts` work?

File: README.md [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
+```bash
+solang compile --target polkadot {path/to/contract}.sol
+```
 
-[price_feed](/price_feed/README.md)
+### Deploying the contract
 
-[psp_pendulum](/price_feed/README.md)
+As a result you will get a file called `{contract}.contract`.
+You can now deploy this contract to a parachain using Contracts UI.
```
Comment (ebma) [inline README.md:29]: I think we can remove this section from this README as it's already covered in the respective READMEs in the sub directories.

**Review by ebma — COMMENTED** [review]
It's not a big deal but let's rename/fix the capitalization of the .sol contracts in [this](https://github.com/pendulum-chain/pendulum-ink-wrapper/blob/38-clean-up-repository-structure/docs/OVERVIEW.md) file as well. They should be called `ERC20Wrapper.sol` and `PriceOracleWrapper.sol`.

I like the name 'Pendulum-Solidity Wrappers' 👍 I'm fine if we change the repo name to `pendulum-solidity-wrappers` then

**Comment by gianfra-t** [comment]
@ebma I think you or someone with access will have to change the name, I don't have proper access for it. But let's go with `Pendulum-Solidity Wrappers`!

**Comment by TorstenStueber** [comment]
I renamed the repo.

@gianfra-t can you change the capitalization [as suggested](https://github.com/pendulum-chain/pendulum-solidity-wrapper/pull/41#pullrequestreview-1963920815)?

**Comment by gianfra-t** [comment]
I modified the capitalization of the contracts in `OVERVIEW`. Is there any other place you are referring to @TorstenStueber ?

**Comment by TorstenStueber** [comment]
Oh, my bad, all good then.

**Review by TorstenStueber — APPROVED** [review]

**Pull request merged**